01-Setup-GreenSock-as-a-Module-with-Parcel.md

```diff
@@ -0,0 +1,56 @@
+# Setup GreenSock As A Module With Parcel
+
+📹 [Video](https://egghead.io/courses/create-amazing-animations-with-greensock)
+
+Start by creating a folder for your new project, then add [parcel bundler](https://parceljs.org/getting_started.html) as a dev dependency
+
+    > yarn add -D parcel-bundler
```

Comment:
This is great. I recommend only using the fenced code blocks -- they're easier and only they support syntax highlighting. Please refer to this [example](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).